cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -191,9 +239,23 @@ object CassandraLauncher {
 
       prepareCassandraDirectory(cassandraDirectory, clean)
 
-      val realHost = host.getOrElse("127.0.0.1")
-      val realPort = if (port == 0) randomPort else port
-      val storagePort = freePort()
+      val realHost = host.getOrElse(DEFAULT_HOST)
```

Comment:
I think there was a flaw here. If the user passes a host, we should use it when binding the ports. See below.